### DISCUSSION TITLE
Is a quarto project different from a regular RStudio project?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#1193] [Q&A] [answered]
If I have a regular RStudio project, is it worth it to replace it with a quarto project or just leave it and adds quarto documents?

**Comment by jjallaire** [comment, marked as answer]
Quarto projects and RStudio projects can co-exists (a directory can be both). If you aren't creating a website or a book the main benefit of Quarto project is shared options/metadata across multiple documents and the ability to define an output directory.

**Comment by ManuelSpinola** [comment]
Thank you very much.

On Tue, 21 Jun 2022 at 10:50 J.J. Allaire ***@***.***> wrote:

> Quarto projects and RStudio projects can co-exists (a directory can be
> both). If you aren't creating a website or a book the main benefit of
> Quarto project is shared options/metadata across multiple documents and the
> ability to define an output directory.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/1193#discussioncomment-2995759>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBYUAPAOAGNXJYREXK3VQHXFXANCNFSM5ZM6GXRA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app
